src/Microsoft.SqlTools.ServiceLayer/Agent/AgentService.cs

```diff
@@ -1211,6 +1219,215 @@ public static XmlDocument CreateJobXmlDocument(string svrName, string jobName)
             return new Tuple<SqlConnectionInfo, DataTable, ServerConnection>(sqlConnInfo, dt, serverConnection);
         }
 
+                // Notebook helper function to execute queires to be later replaced by QueryExecutionService method.
+        public DataSet executeQuery(ConnectionInfo connInfo, string SqlQuery)
```

Comment:
Use xml doc comments for method headers (/** should auto-populate the header if you're using VS)